packages/web3-eth/test/integration/defaults.transactionBlockTimeout.test.ts

```diff
@@ -19,62 +19,60 @@ import { DEFAULT_RETURN_FORMAT } from 'web3-utils';
+import { Web3 } from 'web3';
+import { Web3Account } from 'web3-eth-accounts';
+import { SendTransactionEvents } from '../../src';
 
 import {
 	closeOpenConnection,
-	createTempAccount,
 	getSystemTestProvider,
 	isSocket,
 	itIf,
 	waitForOpenConnection,
+	createLocalAccount,
+	sendFewSampleTxs,
 } from '../fixtures/system_test_utils';
 
-import { sendFewTxesWithoutReceipt } from './helper';
-
 const MAX_32_SIGNED_INTEGER = 2147483647;
+const gas = 21000;
 /* eslint-disable jest/no-standalone-expect */
 describe('defaults', () => {
-	let web3Eth: Web3Eth;
-	let eth2: Web3Eth;
+	let web3: Web3;
 	let clientUrl: string;
+	let account1: Web3Account;
+	let account2: Web3Account;
 
-	beforeEach(() => {
+	beforeEach(async () => {
 		clientUrl = getSystemTestProvider();
-		web3Eth = new Web3Eth(clientUrl);
+		web3 = new Web3(clientUrl);
+		// Make the test run faster by casing the polling to start after 2 blocks
+		web3.eth.transactionBlockTimeout = 2;
+
+		// Increase other timeouts so only `transactionBlockTimeout` would be reached
+		web3.eth.transactionSendTimeout = MAX_32_SIGNED_INTEGER;
+		web3.eth.transactionPollingTimeout = MAX_32_SIGNED_INTEGER;
+		web3.eth.blockHeaderTimeout = MAX_32_SIGNED_INTEGER / 1000;
 	});
 
 	afterEach(async () => {
-		await closeOpenConnection(web3Eth);
-		await closeOpenConnection(eth2);
+		web3.eth.transactionBlockTimeout = 50;
+		await closeOpenConnection(web3.eth);
 	});
 
 	describe('defaults', () => {
 		it('should fail if transaction was not mined within `transactionBlockTimeout` blocks', async () => {
-			const eth = new Web3Eth(clientUrl);
-			const tempAcc1 = await createTempAccount();
-			const tempAcc2 = await createTempAccount();
-
-			// Make the test run faster by casing the polling to start after 2 blocks
-			eth.transactionBlockTimeout = 2;
-
-			// Increase other timeouts so only `transactionBlockTimeout` would be reached
-			eth.transactionSendTimeout = MAX_32_SIGNED_INTEGER;
-			eth.transactionPollingTimeout = MAX_32_SIGNED_INTEGER;
-			eth.blockHeaderTimeout = MAX_32_SIGNED_INTEGER / 1000;
-
-			const from = tempAcc1.address;
-			const to = tempAcc2.address;
-			const value = `0x0`;
-
+			account1 = await createLocalAccount(web3);
+			account2 = await createLocalAccount(web3);
 			// Setting a high `nonce` when sending a transaction, to cause the RPC call to stuck at the Node
 			const sentTx: Web3PromiEvent<
 				TransactionReceipt,
 				SendTransactionEvents<typeof DEFAULT_RETURN_FORMAT>
-			> = eth.sendTransaction({
-				to,
-				value,
-				from,
+			> = web3.eth.sendTransaction({
+				from: account1.address,
+				to: account2.address,
+				gas,
+				value: '0x1',
 				// Give a high nonce so the transaction stuck forever.
 				// However, make this random to be able to run the test many times without receiving an error that indicate submitting the same transaction twice.
 				nonce: Number.MAX_SAFE_INTEGER - Math.floor(Math.random() * 100000000),
```

Comment:
```suggestion
				nonce: Number.MAX_SAFE_INTEGER ,
```
This logic isn't need for this test and others if we're using a new account for each test?

Reply:
it is was done before me

Reply:
done